flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/calls/SearchOperatorGen.scala

```diff
@@ -97,6 +97,9 @@ object SearchOperatorGen {
            |if (!${needle.nullTerm}) {
            |  $resultTerm = $negation$setTerm.contains(${needle.resultTerm});
            |  $nullTerm = !$resultTerm && $setTerm.containsNull();
```

Comment:
If `needle` is not null, the result should never be `unknown`. Why not `$nullTerm = false`?

Reply:
based on logic it seems this should always result to `false` in this `if` branch;
I changed it to make it more clear.

thanks for highlighting